package.json

```diff
@@ -7,34 +7,29 @@
 		"@fortawesome/fontawesome-free": "^5.13.0",
 		"TYPO3CMS-Guide-HowToDocument": "github:TYPO3-Documentation/TYPO3CMS-Guide-HowToDocument",
 		"autocompleter": "^6.0.3",
+		"babel-loader": "^8.2.2",
 		"bootstrap": "^4.4.1",
-		"connect-livereload": "^0.6.1",
-		"grunt": "^1.1.0",
-		"grunt-cli": "^1.3.2",
-		"grunt-contrib-clean": "2.0.0",
-		"grunt-contrib-copy": "1.0.0",
-		"grunt-contrib-uglify": "^4.0.1",
-		"grunt-contrib-watch": "^1.1.0",
-		"grunt-exec": "^3.0.0",
-		"grunt-sass": "^3.1.0",
-		"grunt-stylelint": "^0.15.0",
+		"copy-webpack-plugin": "^8.0.0",
+		"css-loader": "^5.1.2",
+		"file-loader": "^6.2.0",
 		"http-server": "^0.12.3",
 		"jquery": "^3.5.1",
-		"modernizr": "^3.11.0",
-		"node-sass": "^4.14.0",
+		"mini-css-extract-plugin": "^1.3.9",
+		"node-sass": "^5.0.0",
```

Comment:
suggestion: I would prefer for us to use `sass` rather than `node-sass` so we don’t need native bindings.

Reply:
Sure, I'm not available to work on this anymore this week. Could you fix it for me? Thank you!

Reply:
Switched to `sass`!